src/views/RoomDetail.vue

```diff
@@ -141,8 +145,8 @@ export default {
       return isSignedIn
     },
     roomOccupation: function () {
-      if (this.room.checkIns.user) {
-        return this.room.checkIns.user.length
+      if (this.room.checkIns.length > 0) {
```

Comment:
Wieso wird hier die Logik verändert, für das Anzeigen der Liste?

Reply:
Weil es das User Objekt in CheckIn nicht mehr gibt. Es werden dort jetzt die User reingeladen, damit wir keinen duplicated Code haben.